jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/resource/Resource.java

```diff
@@ -55,12 +55,11 @@
  * a file, a URL or an entry in a jar file.
  * </p>
  */
-//TODO remove
+// TODO Resource API should be cleaned up and simplified
```

Comment:
We might not need the `defaultUseCaches` settings any more if we're not using URLs?